app/presenters/brexit_taxon_presenter.rb

```diff
@@ -6,8 +6,9 @@ class BrexitTaxonPresenter
     to: :taxon
   )
 
-  def initialize(taxon)
+  def initialize(taxon, index = false)
     @taxon = taxon
+    @index = index if index
```

Comment:
You could use `index = nil` in the params and remove the conditional on line 11, so `@index = index`